app/helpers/sidebar_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SidebarHelper
+  def menu_active?(name)
```

Comment:
```suggestion
  def menu_active(name)
```

https://rubystyle.guide/#bool-methods-qmark